core/kubernetes/config/config-importer-job.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: config-importer
+spec:
+  template:
+    spec:
+      containers:
+      - args:
+        - sh
+        - -c
+        - wait-on -t 30000 https-get://openhim-core:8080/heartbeat && node openhimConfig.js
```

Comment:
Just a note: This might need to be changed later on when we have the consolidated service names that we will be using. This might even also be a custom domain name setup with Ingress